src/plugins/discover/server/locator/searchsource_from_locator.ts

```diff
@@ -147,7 +148,17 @@ export function searchSourceFromLocatorFactory(services: LocatorServicesDeps) {
 
     // Inject sort
     if (savedSearch.sort) {
-      const sort = getSortForSearchSource(savedSearch.sort as Array<[string, string]>, index);
+      const defaultSortingSetting = await services.uiSettings.get(SORT_DEFAULT_ORDER_SETTING);
+      const uiSettingsSyncReplacement = {
```

Comment:
What's this code doing and does it change the behaviour of reporting at all?

Reply:
@davismcphee It now reads `SORT_DEFAULT_ORDER_SETTING` setting. Before is was implicitly using `desc`.

Reply:
Would you suggest to revert?

cc @kertal